native/src/modules/app/constants/buildConfig.js

```diff
@@ -1,20 +1,23 @@
 // @flow
 
-import type { BuildConfigType } from '../../../../build-configs/BuildConfigType'
-import loadBuildConfig from '../../../../build-configs'
-import integreatAppLogo from '../../../../build-configs/integreat/assets/app-logo.png'
-import integreatLocationMarker from '../../../../build-configs/integreat/assets/location-marker.svg'
-import malteAppLogo from '../../../../build-configs/malte/assets/app-logo.svg'
-import malteLocationMarker from '../../../../build-configs/malte/assets/location-marker.svg'
-import integreatIntroLanguageIcon from '../../../../build-configs/integreat/assets/Language.svg'
-import integreatIntroEventsIcon from '../../../../build-configs/integreat/assets/Events.svg'
-import integreatIntroOffersIcon from '../../../../build-configs/integreat/assets/Offers.svg'
-import integreatIntroSearchIcon from '../../../../build-configs/integreat/assets/Search.svg'
-import malteIntroLanguageIcon from '../../../../build-configs/malte/assets/Language.svg'
-import malteIntroEventsIcon from '../../../../build-configs/malte/assets/Events.svg'
-import malteIntroOffersIcon from '../../../../build-configs/malte/assets/Offers.svg'
-import malteIntroSearchIcon from '../../../../build-configs/malte/assets/Search.svg'
-import { INTEGREAT_ASSETS, MALTE_ASSETS } from '../../../../build-configs/AssetsType'
+import type { BuildConfigType } from 'build-configs/BuildConfigType'
+import loadBuildConfig from 'build-configs'
+import integreatAppLogo from 'build-configs/integreat/assets/app-logo.png'
+import integreatLocationMarker from 'build-configs/integreat/assets/location-marker.svg'
+import malteAppLogo from 'build-configs/malte/assets/app-logo.svg'
+import malteLocationMarker from 'build-configs/malte/assets/location-marker.svg'
+import integreatIntroLanguageIcon from 'build-configs/integreat/assets/Language.svg'
+import integreatIntroEventsIcon from 'build-configs/integreat/assets/Events.svg'
+import integreatIntroOffersIcon from 'build-configs/integreat/assets/Offers.svg'
+import integreatIntroSearchIcon from 'build-configs/integreat/assets/Search.svg'
+import malteIntroLanguageIcon from 'build-configs/malte/assets/Language.svg'
+import malteIntroEventsIcon from 'build-configs/malte/assets/Events.svg'
+import malteIntroOffersIcon from 'build-configs/malte/assets/Offers.svg'
+import malteIntroSearchIcon from 'build-configs/malte/assets/Search.svg'
+import { INTEGREAT_ASSETS, MALTE_ASSETS } from 'build-configs/AssetsType'
+
+export { BuildConfigType, FeatureFlagsType, LocalesType } from 'build-configs/BuildConfigType'
```

Comment:
I think we could also just import them from `'build-configs/BuildConfigType'` in each file. I got confused right now why these types from from within the native project.